misago/markup/tests/test_parser.py

```diff
@@ -192,7 +192,7 @@ def test_clean_current_link(self):
 """.strip()
 
         expected_result = """
-<p>Lorem ipsum: <a href="/" rel="nofollow">test.com</a></p>
```

Comment:
I've discussed this with fellow seo wizard and general opinion in this case is decision between "I want users to vote for internal pages with their links" vs "I don't want users to vote for internal pages with their links". On forum this will be "I would like threads forum users link to frequently have higher ranking within the site than ones that users don't link to."

This makes perfect sense. *The dude abides!*